cli/admin/src/main/java/io/pravega/cli/admin/utils/CLIControllerConfig.java

```diff
@@ -32,29 +32,30 @@
 
     private static final Property<String> CONTROLLER_REST_URI = Property.named("controller.connect.rest.uri", "localhost:9091");
     private static final Property<String> CONTROLLER_GRPC_URI = Property.named("controller.connect.grpc.uri", "localhost:9090");
-    private static final Property<Boolean> AUTH_ENABLED = Property.named("controller.connect.channel.auth", false);
-    private static final Property<String> CONTROLLER_USER_NAME = Property.named("controller.connect.credentials.username", "");
-    private static final Property<String> CONTROLLER_PASSWORD = Property.named("controller.connect.credentials.pwd", "");
-    private static final Property<Boolean> TLS_ENABLED = Property.named("controller.connect.channel.tls", false);
-    private static final Property<String> TRUSTSTORE_JKS = Property.named("controller.connect.trustStore.location", "");
+    private static final Property<Boolean> AUTH_ENABLED = Property.named("channel.auth", true);
```

Comment:
I see this is where the config name changes come from. Why did you remove `controller` out of these? For example `CONTROLLER_USER_NAME` still has `CONTROLLER` in it, but the property name has that `controller` removed from it.

Reply:
I changed the it from `CONTROLLER_USER_NAME` to `USER_NAME`, and same for the password now.